.github/workflows/commit-checker.yml

```diff
@@ -17,11 +17,11 @@ on:
       tool-repository:
         description: Commit Checker ICU repo
         required: true
-        default: 'unicode-org/icu'
+        default: 'srl295/icu'
```

Comment:
@AEApple maybe we don't want to commit this to maint. Would it be too much hassle to type in srl295 each time?

Reply:
Sure, as long as we're sure it's running against maint-41 for the actual commit check and only using the srl925 for the list of commits to skip.